chaincfg/params.go

```diff
@@ -384,6 +363,14 @@ func Register(params *Params) error {
 	return nil
 }
 
+// MustRegister performs the same function as Register except it panics if there
+// is an error.  This should only be called from package init functions.
+func MustRegister(params *Params) {
```

Comment:
I was thinking it would be useful for others to be able to register their custom networks at init time too, but I suppose they could just write something similar.